etor/src/main/resources/openapi_etor.yaml

```diff
@@ -105,6 +105,30 @@ paths:
             application/json:
             schema:
               $ref: '#/components/schemas/NotFoundError'
+  /v1/etor/metadata/summary/{sender}:
+    get:
+      summary: Returns all the order and their status for particular hospital
```

Comment:
Couple of typos: `Returns all the orders and their status for a particular hospital`